app/uk/gov/hmrc/eusubsidycompliancefrontend/models/NonHmrcSubsidy.scala

```diff
@@ -22,7 +22,9 @@ import play.api.libs.json.{Json, OFormat}
 import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.{EORI, EisSubsidyAmendmentType, SubsidyAmount, SubsidyRef, TraderRef}
 
 case class NonHmrcSubsidy(
-  subsidyUsageTransactionID: Option[SubsidyRef], // TODO if we can't get SCP09 sorted we'll need to uppercase ID in the format
+  subsidyUsageTransactionID: Option[
```

Comment:
I think this was better on one line.

Reply:
I think it would have been in one line if TODO was not there, will try to move TODO to next line